pom.xml

```diff
@@ -24,7 +24,7 @@
         <jackson.version>2.7.4</jackson.version>
         <guava.version>19.0</guava.version>
         <powermock.version>1.7.0RC2</powermock.version>
-        <azuresdk.version>1.10.0</azuresdk.version>
+        <azuresdk.version>1.13.0</azuresdk.version>
```

Comment:
does this need update in azure-common plugin?

Reply:
no, sync with xiangyang, only adding new dependencies should consider azure-commons plugin.